packages/orchestration/src/orchestration-api.ts

```diff
@@ -118,10 +136,13 @@ export interface Chain<CI extends ChainInfo> {
 
   // "makeAccount" suggests an operation within a vat
   /**
+   * TODO: scope to `icaEnabled: true`;
    * Creates a new Orchestration Account on the current Chain.
    * @returns an object that controls the account
    */
-  makeAccount: () => Promise<OrchestrationAccount<CI>>;
+  makeAccount: CI extends { namespace: 'cosmos' }
```

Comment:
consider handling this condition within `OrchestrationAccount`

Reply:
The idea was to constrain calling `makeAccount()`, so I thought here was more appropriate.

I considered:
```ts
export type OrchestrationAccount<CI extends Partial<ChainInfo>> = CI extends {
  chainId: string;
}
  ? OrchestrationAccountCommon &
      (CI['chainId'] extends `agoric${string}`
        ? LocalAccountMethods
        : CosmosChainAccountMethods<CI>)
  : never;
```
But things (mainly `orchestrator.js`) seem to break due to the `never`. 

This seems like it's working as expected:
```ts
  makeAccount: () => CI extends { chainId: string }
    ? Promise<OrchestrationAccount<CI>>
    : never;
```

Note: I left a TODO to further constrain by `{ icaEnabled: true }`. With the current CI generic, this didn't seem worth the refactor cost atm.